Utilities/Extensions/Prime.cs

```diff
@@ -0,0 +1,102 @@
+
+using System.Collections.Generic;
+
+namespace PrimeCalculations
+{
+    public static class PrimeCalc
+    {
+        //--------------------------------------------------------------------
+        // IsPrime
+        //--------------------------------------------------------------------
+
+        /// <summary>
+        /// Indicates whether the integer given is a prime number
+        /// </summary>
+        public static bool IsPrime(this int candidate)
+        {
+            if ((candidate & 1) == 0)
+            {
+                if (candidate == 2)

```

Comment:
Codacy found an issue: [Use the '?:' operator here.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6357239)